spec/resources/coins/coin_decorator_spec.rb

```diff
@@ -20,6 +21,12 @@
     end
   end
 
+  describe "#artists" do
+    it "renders the linked artists" do
+      expect(decorator.artists).to eq(["artist person artist role"])
```

Comment:
Do you think it would be better to separate the role with a comma, or put it in parentheses?

Reply:
I'll add a comma